static/js/composebox_typeahead.js

```diff
@@ -841,6 +857,54 @@ exports.content_typeahead_selected = function (item, event) {
         // with the topic and the final **.
         const start = beginning.length - this.token.length;
         beginning = beginning.substring(0, start) + item + '** ';
+    } else if (this.completing === 'time_jump') {
+        const flatpickr_input = $("<input id='#timestamp_flatpickr'>");
```

Comment:
Is the right way to do this to dynamically generate an input, or should it live in the compose box DOM somewhere?

I guess we should also be sure to test this with message-edit UI.